### ISSUE TITLE
LUA Error when using item from the quest tracker

### ISSUE THREAD

**Issue by KNP54** [Jaliborc/PetTracker#395]
> :warning: Due to the nature of taint, errors can easily be blamed by the client on the wrong addon. When running multiple ones, it is generally not clear which caused the issue. Tracking down the cause can also be complicated.

**Which software were you running?**
 - Addon version: 11.0.1
 - Client used: Retail 11.0.2.56162
**Please describe how to reproduce it.**
1. Accept a quest which requires using an item
2. Click on that item from the quest tracker

1x [ADDON_ACTION_FORBIDDEN] AddOn 'PetTracker' tried to call the protected function 'UseQuestLogSpecialItem()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `UseQuestLogSpecialItem'
[string "@Blizzard_ObjectiveTracker/Blizzard_ObjectiveTrackerShared.lua"]:94: in function <...ObjectiveTracker/Blizzard_ObjectiveTrackerShared.lua:87>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_FORBIDDEN"
events = <table> {
}

 **Have you tried disabling all other addons and verified the problem remains?**
 No. But disabling Pet Tracker fixes the issue.

**Comment by KNP54** [comment]
Here is the taint log:

8/20 05:49:32.631  Interface/AddOns/Blizzard_ObjectiveTracker/Blizzard_ObjectiveTrackerShared.lua:94
8/20 05:49:32.631  An action was blocked because of taint from PetTracker - UseQuestLogSpecialItem()
8/20 05:49:32.631      Interface/AddOns/Blizzard_ObjectiveTracker/Blizzard_ObjectiveTrackerShared.lua:94

**Comment by 1Holy-Z** [comment]
I am experiencing this as well.

1x [ADDON_ACTION_FORBIDDEN] AddOn 'PetTracker' tried to call the protected function 'UseQuestLogSpecialItem()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `UseQuestLogSpecialItem'
[string "@Blizzard_ObjectiveTracker/Blizzard_ObjectiveTrackerShared.lua"]:94: in function <...ObjectiveTracker/Blizzard_ObjectiveTrackerShared.lua:87>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_FORBIDDEN"
events = <table> {
}

**Comment by Gaviin1242** [comment]
Yeah same problem here.  Guess I'll disable the addon until this gets fixed.

**Comment by CrusaderEsper** [comment]
Error doesn't happen after disabling PetTracker. Might be a taint issue but it does seem to be the cause here.

**Comment by Andf** [comment]
Same issue here, I had to disable the addon:

2x [ADDON_ACTION_FORBIDDEN] AddOn 'PetTracker' tried to call the protected function 'UseQuestLogSpecialItem()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: in function `UseQuestLogSpecialItem'
[string "@Blizzard_ObjectiveTracker/Blizzard_ObjectiveTrackerShared.lua"]:94: in function <...ObjectiveTracker/Blizzard_ObjectiveTrackerShared.lua:87>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_FORBIDDEN"
events = <table> {
}

**Comment by nancikennedy** [comment]
I am also having the issue

**Comment by Synric101** [comment]
Also having this problem

**Comment by Cl3tus** [comment]
Same, disabling PetTracker solves the issue

**Comment by Thom357** [comment]
I'm having the same issue, ended up disabling pettracker as well.
I noticed it is also missing data for the war within.
I have switched to another addon called battle pet completionlist, which doesn't have these errors, and already have the war within pets on the map.
